### DISCUSSION TITLE
Welcome to partylan Discussions!

### DISCUSSION THREAD

**Discussion by gyf304** [gyf304/partylan#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mark522412** [comment]
hi. this is removed from steam. why not make a standalone app?
obviously steam going to remove it again. you know why? Because these type of software can be used to play pirated games with lan support. that's why discord left their app as well as Hamachi. radmin vpn still works but because it uses their server its always so busy and laggy. serverless p2p apps have the best speed in my opinion. that is why im asking to release a stanalone version. but if you looking to sell it which is understandable you can add a license code or something its up to you. if you release it on steam they will come up with another excuse to remove it. im just saying

**Comment by aos-antos** [comment]
I know maybe it's not possible, but hear me out. Maybe you could try to launch it on GOG in some sort of way. They have a bunch of old games with lan that don't have any sort of multiplayer function active at the moment, and if for some reason you could work with them for a integration inside the software for lan multiplayer. I'm not tech savvy or anything, but it would be cool.

**Comment by 77155331** [comment]
你好，我是中国使用者，在使用中发现只能通过IP链接才可以实现组网，问题出现在没有MAC地址，是否可以改进一下自定义MAC地址和自定义语言文件这两个功能？

**Comment by RishiSahoo099** [comment]
can i play REPO game with this mod?? , me and my friend are haaving some serious ping issue
